test/integration/scheduler/predicates_test.go

```diff
@@ -870,3 +870,68 @@ func TestInterPodAffinity(t *testing.T) {
 		}
 	}
 }
+
+// TestNodePIDPressue verifies that scheduler's CheckNodePIDPressurePredicate predicate
+// functions works correctly.
+func TestNodePIDPressue(t *testing.T) {
```

Comment:
s/TestNodePIDPressue/TestNodePIDPressure/
Pleaes fix the comment as well.

Reply:
done